pyinteraph/file_validation.py

```diff
@@ -0,0 +1,15 @@
+import argparse
+import os
+
+
+class ArgumentParserFileExtensionValidation(argparse.FileType):
```

Comment:
I think this can just be defined in the `reformat_interaction_network_graph` module since it's only used there. If we end up using in other places we can consider taking it out as a e.g. `utils` module or similar